qa/tasks/ceph_manager.py

```diff
@@ -1363,7 +1363,6 @@ def __init__(self, controller, ctx=None, config=None, logger=None,
         self.controller = controller
         self.next_pool_id = 0
         self.cluster = cluster
-        self.cephadm = cephadm
```

Comment:
no need to move this

Reply:
This variable too is being overriden; see https://github.com/ceph/ceph/pull/38481/files#diff-203644a2b9fd4a8cf54b0c52970ffbd1aebb87a134dcf9fc0dc88286da7d4ab6R1380. So, I moved it to the bunch where variables are being overriden in a subclass and added a comment; see https://github.com/ceph/ceph/pull/38481/files#diff-c0b3c977c8b56ffb318d67c2f39c109d3a097714adff0f7870217cabe79e2486R785.